src/components/Chip/index.tsx

```diff
@@ -1,13 +1,21 @@
 import React from 'react';
+import type { MouseEventHandler } from 'react';
 
 import { Container, ChipContent } from './Chip.styles';
 
+interface Props {
+  children: React.ReactNode;
+  className: string;
```

Comment:
className도 optinal해야 할 것 같습니다!!
```suggestion
  className?: string;
```